.github/workflows/automerge.yml

```diff
@@ -0,0 +1,12 @@
+name: automerge
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '42 2,5,8,11 * * *'
+
+jobs:
+  automerge:
+    uses: salesforcecli/github-workflows/.github/workflows/automerge.yml@main
```

Comment:
The `run:` for `!inputs.skipCI` and `inputs.skipCI` are exactly the same. Looks like you'll need to add `--skips-ci` ([orb](https://github.com/forcedotcom/npm-release-management-orb/blob/cab96c72fa3fc12364ad38c613281e56e3c92659/src/jobs/dependabot-automerge.yml#L42))